cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,37 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SettingsPageObject from "../support/pages/settings.pageObject";
+import HomePageObject from "../support/pages/home.pageObject";
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+
 describe('User', () => {
-  before(() => {
+  let user;
+  let anotherUser;
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.register(user.email, user.username, user.password);
   });
 
-  it.skip('should be able to follow the another user', () => {
+  it('should be able to follow the another user', () => {
+    settingsPage.visit();
+
+    settingsPage.clickOnLogOutBtn();
 
+    cy.task('generateAnotherUser').then((generateAnotherUser) => {
+      anotherUser = generateAnotherUser;
+    });
+    cy.register(anotherUser.email, anotherUser.username, anotherUser.password);
```

Comment:
Consider using the page object methods to simulate user registration instead of directly using `cy.register`. This will better align with testing the UI process.